src/main/kotlin/no/nav/aareg/teknisk_historikk/aareg_services/AaregServicesKonsument.kt

```diff
@@ -34,7 +37,8 @@ data class AaregServicesConfig(
 class AaregServicesConsumer(
     private val aaregServicesConfig: AaregServicesConfig,
     private val azureTokenConsumer: AzureTokenConsumer,
-    private val restTemplate: RestTemplate
+    private val restTemplate: RestTemplate,
+    @Value("\${app.name}") private val appName: String
```

Comment:
`\` for mye? ikke helt stødig på kotlin men den så out of place ut

Reply:
Kotlin har templating-støtte for strings, så vi må escape ut av det for å ta i bruk Spring sin templating 🙈